packages/rlc-codegen/src/helpers/operationHelpers.ts

```diff
@@ -60,18 +60,22 @@ export function getPathParamDefinitions(
 }
 
 export function hasPagingOperations(model: RLCModel) {
-  for (const [path, details] of Object.entries(model.paths)) {
-    if (Boolean(details.annotations?.isPageable)) {
-      return true;
+  for (const details of Object.values(model.paths)) {
+    for (const method of Object.values(details.methods)) {
+      if (Boolean(method[0].annotations?.isPageable)) {
+        return true;
+      }
     }
   }
   return false;
 }
 
 export function hasPollingOperations(model: RLCModel) {
-  for (const [path, details] of Object.entries(model.paths)) {
-    if (Boolean(details.annotations?.isLongRunning)) {
-      return true;
+  for (const details of Object.values(model.paths)) {
+    for (const method of Object.values(details.methods)) {
+      if (Boolean(method[0].annotations?.isLongRunning)) {
+        return true;
+      }
```

Comment:
I wonder whether we could put the result info RLCModel, and then get the value from there directly ?

Reply:
Let me clarify my understanding, you mean we could 

1. Transform data to RLCModel
2. Then calculate derived data like hasLRO, hasPaging from other data in RLCModel
3. Save derived data back to RLCModel so we could have less calculation

Reply:
no, I mean we can have an annotation in RLCModel directly to tell us whether there's any lro or paging in the whole library. this annontation value can be get while we do other transforms for operation level transforms. Then in the generated code, we don't need to go through all the paths or operations to find out.